### ISSUE TITLE
Crashes: ArcMap 10.4.1

### ISSUE THREAD

**Issue by jwhayes76** [WSDOT-GIS/arcmap-spellcheck#1]
Jeff,

Love that you made a spell checker.  The only problem is it crashes ArcMap whenever I push the button.  I tried it in layout and data view.  I also tried it while I has a layerfile's name highlighted. I have Microsoft Word 2010. Any suggestions?

Joe

**Comment by JeffJacobson** [comment]
Do you have Microsoft Word installed? (It's using Word's spellchecker) If so, what version?
What exactly happens when it crashes? E.g., are there any error messages, and if so, what do they say?

**Comment by jwhayes76** [comment]
Yes, Word 2010 installed.  No error messages, it just crashed and gives me the ESRI window saying "ArcGIS for Desktop has encountered a serious application error..."  I'm using Windows 7 64.

**Comment by jwhayes76** [comment]
[ArcMap_10.4.1.5686_0_07_13_2017_09_41_16.txt](https://github.com/WSDOT-GIS/arcmap-spellcheck/files/1145988/ArcMap_10.4.1.5686_0_07_13_2017_09_41_16.txt)

Here's the dump file.  Change the extension back to .dmp

**Comment by JeffJacobson** [comment]
We're currently using Windows 10 (64-bit) and Word 2016. I've updated the [README.md](../blob/master/README.md) file to note that the addin has only been tested with the 2016 version of Word.

At some point we want to get rid of the dependency on Word and instead use an open-source spellchecking library. (See issue #3.) 

When I get some time I'll look at the dump file to see if I can learn anything from it.

**Comment by pmacMaps** [comment]
I recently upgraded from ArcGIS 10.3.1 to ArcGIS 10.5.1 on Windows 7.  I had attempted to install this plugin at 10.3.1 (unsupported), but it appeared as an add-in after the upgrade.

The days following the upgrade, ArcMap would crash when trying to open (however, ArcCatalog did open).  After deleting the Add In file in C:\Users\[user]\AppData\Roaming\ESRI\Desktop10.5, ArcMap opened fine.

I'm guessing it may have been an issue with this add-in, although I don't know for sure.
